TransformationSystem/Client/TransformationClient.py

```diff
@@ -2,24 +2,16 @@
 
```

Comment:
I did not look in detail, but I know that many tests exist for the transformation system. Did you have a look whether it is needed to adapt them ?